contribution.md

```diff
@@ -0,0 +1,245 @@
+# How to contribute to OMZ
```

Comment:
This file should be named `CONTRIBUTING.md` to enable [GitHub integration](https://help.github.com/en/articles/setting-guidelines-for-repository-contributors).

Reply:
Done